libs/ledgerjs/packages/types-live/src/operation.ts

```diff
@@ -11,6 +11,8 @@ export type OperationType =
   | "NONE"
   | "CREATE"
   | "REVEAL"
+  // APTOS
+  | "UNKNOWN"
```

Comment:
from a user perspective, i wonder what an "UNKNOWN" means, what is the semantic of this and also why would it be specific to APTOS?